boards/nxp/frdm_mcxn947/board.c

```diff
@@ -14,6 +14,7 @@
 /* System clock frequency. */
 extern uint32_t SystemCoreClock;
 
+#ifndef CONFIG_BOOTLOADER_MCUBOOT
```

Comment:
This seems like an option that is not directly mcuboot related. This option should get a clear name (ENABLE_LPCAC ?) and be enabled if `!BOOTLOADER_MCUBOOT`

Reply:
A lot of the board init code is executed in mcuboot. This avoids re-running this, I don't see a need for a separate Kconfig for this.

Reply:
OK,

I'm only pointing out that another bootloader or a zephyr application that is starting another application could just as well be in the same situation. Directly using `CONFIG_BOOTLOADER_MCUBOOT` limits the usability.

Reply:
I will look to update this code as part of a separate PR where we can remove this dependency on `CONFIG_BOOTLOADER_MCUBOOT`

Reply:
Could we base this on the state of `CONFIG_INIT_ARCH_HW_AT_BOOT`? A bootloader would typically select this to indicate the image is being chainloaded

Reply:
@danieldegrasse @Laczen. I have removed the `CONFIG_BOOTLOADER_MCUBOOT`. I will reintroduce this check in the FlexSPI PR, this check was intended to prevent reconfiguring the PLL which is used to clock the FlexSPI.

Reply:
@danieldegrasse, can you please re-review. PR has been updated with your feedback.